src/test/java/ludogorie_soft/reservations_platform_api/service/impl/BookingServiceImplTest.java

```diff
@@ -288,4 +311,77 @@ void testEditBookingShouldThrowWhenTheDatesAreNotAvailable() throws IOException,
         assertThrows(NotAvailableDatesException.class, () -> bookingService.editBooking(createdBooking.getId(), editRequestDto));
     }
 
+    @Test
+    public void testAddCustomerDataToBooking_CustomerExists() {
```

Comment:
Methods in test classes should NOT be public. Please check Sonar suggestions for code smells https://sonarcloud.io/summary/new_code?id=Ludogorie-Soft_reservations-platform-be&pullRequest=19

Reply:
fixed